specification/monitor/data-plane/Microsoft.Insights/preview/2023-03-01-preview/examples/MetricBatchExample.json

```diff
@@ -0,0 +1,520 @@
+{
+  "parameters": {
+    "baseUrl": "https://eastus.metrics.monitor.azure.com/",
+    "subscriptionId": "123456728-abcd-1234-abcd-123456789abc",
+    "timespan": "2020-08-01T20:00:00.123Z/2020-08-01T23:00:00.345Z",
+    "interval": "PT15M",
+    "metricnamespace": "Microsoft.Storage/storageAccounts",
+    "metricnames": "Ingress,Egress",
+    "aggregation": "average,minimum,maximum",
+    "top": 10,
+    "orderby": "average desc",
+    "$filter": "ApiName eq 'GetBlob' or ApiName eq 'GetBlobServiceProperties' and Authentication eq '*'",
+    "region": "eastus",
```

Comment:
I'm not sure it makes sense to continue having a region param when we're requiring it to be encoded in the hostname?

Reply:
Will remove in the next commit